Style/login.css

```
@@ -0,0 +1,105 @@
+form{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 40px;
+  text-align: center;
+  margin-top: 100px;
+}
+
+label {
+  display: block;
+  margin: 16px 0 8px;
+  text-align: left;
+
+}
+
+input {
+  display: block;
+  padding: 16px 24px;
+  margin: 16px 0px;
+  width: 640px;
+  height: 56px;
+  box-sizing: border-box;
+  border: 0px;
+  border-radius: 12px;
+  background-color: #F3F4F6;
+}
+
+button {
+  border: none;
+  display: block;
+  padding: 16px;
+  text-align: center;
+}
+
+#login {
+  width: 396px;
+  height: 132px;
+
+}
+
+.container {
+  display: block;
+}
+
+.emailpass {
+  margin: 8px 0px;
+  font-size: 16px;
+  font-weight: 400;
+  color: #2c2c2c;
+}
+
+#loginb {
```

❗️ 수정요청
변수나 클래스명, 이미지명과 같은 이름은 해당 파일이 어떤 역할을 하는 친구인지 나타내는 큰 단서가 되고 코드파악 및 유지보수에 중요한 역할을 합니다. 가능하면 유의미한 이름을 사용해주세요~

```suggestion
#loginButton {
```